src/qwed_new/api/main.py

```
@@ -583,6 +583,99 @@ async def verify_image(
+            "verified": False
+        }
+
+class ProcessVerifyRequest(BaseModel):
+    trace: str
+    mode: str = "irac"
+    milestones: Optional[list[str]] = None
+
+@app.post("/verify/process")
+async def verify_process(
+    request: ProcessVerifyRequest,
+    tenant: TenantContext = Depends(get_current_tenant),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)